src/lib/openapi/PropertySchema.php

```diff
@@ -269,6 +269,22 @@ public function isRefPointerToSchema():bool
 
     public function isRefPointerToSelf():bool
     {
+        $schema = Json::decode(Json::encode($this->schema->getSchema()->getSerializableData()));
```

Comment:
why do you do this? it is potentially very slow operation.

Reply:
Fixed in c1402a964f781ff222355bfe43dda074cd8b80c9